SWIFT_PACKAGE_MANAGER.md

```diff
@@ -44,7 +46,7 @@ Make sure to select your app's target in the _Provide build settings from_ drop-
 
 ### BraintreeThreeDSecure
 
-Currently, to use the `BraintreeThreeDSecure` library via SPM, you must manually include `CardinalMobile.framework`. Once we receive an xcframework version of CardinalMobile, these steps will no longer be required.
+~Currently, to use the `BraintreeThreeDSecure` library via SPM, you must manually include `CardinalMobile.framework`. Once we receive an xcframework version of CardinalMobile, these steps will no longer be required.~ These steps are only required for versions 5.0.0 to 5.2.0. For versions _greater than_ 5.2.0, you can include CardinalMobile via Swift Package Manager.
```

Comment:
I feel like crossing this statement out is confusing since above we are directing people to this section if they are using version 5.0.0-5.2.0

Reply:
Hmmm... I can rework this section.

Reply:
@sarahkoop Updated! Could you take a look and let me know if my changes make sense?

Reply:
I think this is very clear now 👍

Reply:
Awesome, thanks for the feedback!